compiler-utils/src/main/java/com/squareup/anvil/compiler/internal/ClassReference.kt

```diff
@@ -81,6 +83,15 @@ public fun KtClassOrObject.toClassReference(): Psi {
   return Psi(this, toClassId())
 }
 
+@ExperimentalAnvilApi
+public fun FqName.toClassReference(module: ModuleDescriptor): ClassReference? {
```

Comment:
This is very similar to the `ModuleDescriptor.classReferenceOrNull()` function above.  The key difference would seem to be that one gives priority to `ClassDescriptor` and the other `KtClassOrObject`.

Would it be a good idea to delegate one of these functions to the other?

Reply:
It spotted that literally this moment. I find the extension on `ModuleDescriptor` much harder to find (that's why I created the new extension function). I'll merge them.

Reply:
I've created a separate commit for this. I think this change makes a lot of sense.

Reply:
> I find the extension on `ModuleDescriptor` much harder to find

Yeah, I'd say this is a pretty good indicator that it wasn't very discoverable. 😁   This is a good change. 👍